Assets/Scripts/UI/InGame/Graphics/GraphicsController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Newtonsoft.Json;
+
+namespace Assets.Scripts.UI.InGame
+{
+	public class GraphicsController : MonoBehaviour {
+
+		public GeneralGraphics GeneralGraphic;
+		public QualitySwitcher QualitySwitcher;
+		public ResolutionSwitcher ResolutionSwitcher;
+		public Text label;
+		public Toggle CustomSettings;
+
+
+		private void Update() {
```

Comment:
Do we really need to check this every frame? This logic could also be applied on the moment we mark the checkbox for advanced graphics